### PR TITLE
Enable collecting coverage on the frontend

### PR DESCRIPTION
**Pull request by syucream**
To visualize code coverage for the frontend with codecov, like the AirOne core.

### CODE REVIEW

File: .github/workflows/build-frontend.yml [hunk 1]
```diff
@@ -70,3 +70,9 @@ jobs:
           NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
       - name: test
         run: npm run test
+      - name: codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{secrets.CODECOV_TOKEN}}
```
Comment (syucream) [inline .github/workflows/build-frontend.yml:76]: It needs that owner's operation setting codecov token on GitHub Actions settings. Until the setting, it should fails so I set `continue-on-error: true` for now.

Comment (syucream) [reply .github/workflows/build-frontend.yml:76]: Hmm it possibly requires administrative operation on codecov side ...? 🤔 
https://github.com/dmm-com/airone/pull/1061#issuecomment-1913072630

**Review by userlocalhost — APPROVED** [review]
Thank you for updating it 👍 But I couldn't configure it by my authorization level ...
スクリーンショット 2024-01-29 15 47 19

**Review by hinashi — APPROVED** [review]
LGTM!

**Pull request merged**